### DISCUSSION TITLE
Custom Categories

### DISCUSSION THREAD

**Discussion by gccollect** [ghostfolio/ghostfolio#3213] [Ideas]
Thanks for the great work so far. One feature which would be useful is the ability to have custom categories, similar to what is done in Portfolio Performance with "Taxonomies" where you can define a custom list of categories and assign your holdings to each one. This is particularly useful when you are pursuing a target asset allocation across multiple accounts/pensions which you regularly need to rebalance (e.g. Global Equities, UK Bonds, Properties, Commodities, etc.). Each account may use different funds/ETFs due to availability, but represent essentially the same exposure (e.g. Vanguard FTSE All-World ETF vs iShares World Equity Index Fund). By grouping similar funds/ETFs into the same category, you could then see your allocation to each category in the Allocations page, similar to what you can already get for countries and sectors.

**Comment by dtslvr** [comment]
Can you use tags to achieve this and filter by them?

**Reply by gccollect** [reply]
Aren't tags for activities rather than holdings?

**Reply by dtslvr** [reply]
Holding is a synthetic concept, basically a collection of activities. Therefore, you can currently assign tags at activity level.

**Reply by gccollect** [reply]
I should have said market data instead of holdings as that's the level at which the custom category applies

**Comment by gccollect** [comment]
Is it possible to modify the asset subclass list?

**Reply by dtslvr** [reply]
Asset sub classes are defined [here](https://github.com/ghostfolio/ghostfolio/blob/main/prisma/schema.prisma#L258). You are welcome to suggest an addition in a pull request.
